qiskit/algorithms/amplitude_estimators/mlae.py

```diff
@@ -13,7 +13,7 @@
 """The Maximum Likelihood Amplitude Estimation algorithm."""
 
 from __future__ import annotations
-import typing
+from typing import Sequence, Callable, List, Tuple
```

Comment:
Looks like older python do not allow using those in type variable definition even with future import:
```
$ python3
Python 3.8.10 (default, Nov 14 2022, 12:59:47) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from __future__ import annotations
>>> new_type = list[int]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable
```
not sure why.

Reply:
I think this is true, the issue is only with declarations:
https://peps.python.org/pep-0585/#backwards-compatibility
> On the source level, the newly described functionality requires Python 3.9. For use cases restricted to type annotations, Python files with the “annotations” future-import (available since Python 3.7) can parameterize standard collections, including builtins.